agent/command/git.go

```diff
@@ -169,21 +169,11 @@ func getProjectMethodAndToken(ctx context.Context, comm client.Communicator, td
 		return evergreen.CloneMethodLegacySSH, "", err
 	}
 
-	switch conf.GetCloneMethod() {
-	// No clone method specified is equivalent to using legacy SSH.
-	case "", evergreen.CloneMethodLegacySSH:
-		return evergreen.CloneMethodLegacySSH, token, nil
-	case evergreen.CloneMethodOAuth:
-		if token == "" {
-			return evergreen.CloneMethodLegacySSH, "", errors.New("cannot clone using OAuth if explicit token from parameter and global token are both empty")
-		}
-		token, err := parseToken(globalToken)
-		return evergreen.CloneMethodOAuth, token, err
-	case evergreen.CloneMethodAccessToken:
-		return evergreen.CloneMethodLegacySSH, "", errors.New("cannot specify clone method access token")
+	if token == "" {
+		return evergreen.CloneMethodLegacySSH, "", errors.New("cannot clone using OAuth if explicit token from parameter and global token are both empty")
```

Comment:
On a related note - I can't comment on `globals.go` since it's unchanged, but can we move the clone methods and `ValidateCloneMethod` to be unexported variables/functions in this file, instead of keeping them in the globals? Since the distro setting is gone, it's only used to switch on using an access vs OAuth token in this one command now.

Reply:
yes